### PR TITLE
Break out styles

### PR DESCRIPTION
**Pull request by JoshMock**
Quick and easy. 🧹

### CODE REVIEW

File: resources/web/style/comments.pcss [hunk 1]
```diff
@@ -0,0 +1,10 @@
+p.remark {
```
Comment (nik9000) [inline resources/web/style/comments.pcss]: I don't *think* we actually use these:

```
$ pwd
/home/manybubbles/Downloads/built-docs/html
$ find . -name '*.html' -exec grep --color '[ "]remark[ "]' {} \+
<nothing>
```

File: resources/web/style/guide_base_styles.pcss [hunk 2]
```diff
@@ -0,0 +1,25 @@
+#guide {
```
Comment (nik9000) [inline resources/web/style/guide_base_styles.pcss]: maybe just `base`? everything in the docs is "guide".

File: resources/web/style/guide_base_styles.pcss [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+    code {
+      font-size: 16px;
+    }
+  }
+
+  dd {
+    margin: 0 0 0.5em 2em;
+  }
+
+  ul.lists li {
```
Comment (nik9000) [inline resources/web/style/guide_base_styles.pcss]: I think we have `list.pcss` for this one. It'd probably be worth tracking down if we actually use it though.

File: resources/web/style/js_api.pcss [hunk 4]
```diff
@@ -0,0 +1,33 @@
+/* JS API TOC */
```
Comment (nik9000) [inline resources/web/style/js_api.pcss]: I don't think we use any of these:
```
$ find . -name '*.html' -exec grep --color 'js-api-method' {} \+
<nothing>
$ find . -name '*.html' -exec grep --color 'js-client-docs' {} \+
<nothing>
````

File: resources/web/style/comments.pcss (hunk 1 above)
Comment (JoshMock) [reply resources/web/style/comments.pcss]: removed

File: resources/web/style/guide_base_styles.pcss (hunk 2 above)
Comment (JoshMock) [reply resources/web/style/guide_base_styles.pcss]: renamed!

File: resources/web/style/guide_base_styles.pcss (hunk 3 above)
Comment (JoshMock) [reply resources/web/style/guide_base_styles.pcss]: I scanned and couldn't find any references, so it's dropped now.

File: resources/web/style/js_api.pcss (hunk 4 above)
Comment (JoshMock) [reply resources/web/style/js_api.pcss]: Removed this whole file!

**Pull request merged**